go-apps/meep-metrics-engine/server/routers.go

```diff
@@ -58,7 +58,11 @@ func NewRouter() *mux.Router {
 }
```

Comment:
Route requests to correct version-specific functions.
Update function names after code generation.

Reply:
fixed